CHANGELOG.md

```diff
@@ -464,7 +466,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - Official Docker image is available at:
   https://hub.docker.com/r/fundocker/arnold/
 
-[unreleased]: https://github.com/openfun/arnold/compare/v3.2.0...master
+[unreleased]: https://github.com/openfun/arnold/compare/v3.3.0...master
+[3.2.0]: https://github.com/openfun/arnold/compare/v3.2.0...v3.3.0
```

Comment:
```suggestion
[3.3.0]: https://github.com/openfun/arnold/compare/v3.2.0...v3.3.0
```

Reply:
@lunika fixed and ready for further review!